raiden/settings.py

```diff
@@ -70,6 +70,11 @@
 DEFAULT_WAIT_BEFORE_LOCK_REMOVAL = BlockTimeout(2 * DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS)
 DEFAULT_CHANNEL_SYNC_TIMEOUT = 5
 
+DEFAULT_AVERAGE_BLOCK_TIME = 7.0
+DEFAULT_TIMEOUT_BEFORE_BLOCK_PRUNED = (
+    DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS - 2
+) * DEFAULT_AVERAGE_BLOCK_TIME
```

Comment:
I don't quite understand this calculation.
Putting in the values we get `(5 - 2) * 7 == 21`. 

21s seems like a *very* conservative estimate considering the default pruning cutoffs for geth and parity are 128 and 64 blocks.

`64 * 13s = 832s ~= 13min` (even with the assumed 7s block time we're still at 7+ minutes).

Reply:
Yes, it is quite conservative, the pruning values are configurable and we don't have introspection in the current settings of the clients.

I'd like to err on doing frequent updates instead of having flakiness in the client. Updating every 21 seconds should not be that bad (although I'm updating the average block time because of the other comment, but I think I'm also going to divide the  `DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS` by half.

Reply:
wait, I mixed up values, for some reason I had in mind that the code was using these constants:

```
STATE_PRUNING_AFTER_BLOCKS = 64
STATE_PRUNING_SAFETY_MARGIN = 8
```

I will just update it to `(STATE_PRUNING_AFTER_BLOCKS - STATE_PRUNING_SAFETY_MARGIN) * DEFAULT_AVERAGE_BLOCK_TIME` which would be `840`

Reply:
Ok that makes sense.

Although I think this may be going slightly too far into the other direction.
On mainnet we have 13s blocktime. `64 * 13s = 832` which is lower than the `(64 - 8) * 15 = 840`.

Reply:
what number do you want so that we can merge this?

Reply:
How about 2/3 of the 64 blocks?
That would be 42 blocks.

Reply:
can you please approve the PR?